tests/e2e/authz/grpc_test.go

```diff
@@ -2,6 +2,8 @@ package authz
 
```

Comment:
We should add the builds tags, in each files, now that `cli_test.go` is deleted.

Reply:
> lgtm, one nit!
> 
> Have you noticed a performance bump/hit due to the re-creation of the fixtures for each test? Or the parallelization lower that?


So it turns out it significantly increase in the test running time. It has been decided to do not migrate everything to away testify suite, given that we'll create/use an e2e framework instead.
We can focus on small suite to remove at first, which will instead make us cut our CI time: https://github.com/cosmos/cosmos-sdk/issues/14561